### PR TITLE
added docstrings examples for general:unstack

### LINKED ISSUE #2530: unstack

### PR DESCRIPTION
**Pull request by kunwarsaaim**
Resolves #2530

### CODE REVIEW

**Comment by kunwarsaaim** [comment]
@JLU-Neal Hi, can you have a look at my PR.

File: ivy/array/general.py [hunk 1]
```diff
@@ -99,7 +103,20 @@ def unstack(self: ivy.Array, axis: int, keepdims: bool = False) -> ivy.Array:
         -------
         ret
             List of arrays, unpacked along specified dimensions.
+        
+        Examples
+        --------
+        >>> x = ivy.array([[1, 2], [3, 4]])
+        >>> y = x.unstack(axis=0)
+        >>> print(y)
+        [ivy.array([1, 3]), ivy.array([2, 4])]
```
Comment (JLU-Neal) [inline ivy/array/general.py:112]: I think the correct output here should be:
`
[ivy.array([1,2]),ivy.array([3,4])]
`

File: ivy/array/general.py [hunk 2]
```diff
@@ -99,7 +103,20 @@ def unstack(self: ivy.Array, axis: int, keepdims: bool = False) -> ivy.Array:
...
 
+        >>> x = ivy.array([[1, 2], [3, 4]])
+        >>> y = x.unstack(axis=1, keepdims=True)
+        >>> print(y)
+        [ivy.array([[1], 
+                    [3]]), ivy.array([[2], 
+                    [4]])]
```
Comment (JLU-Neal) [inline ivy/array/general.py:119]: I think the correct output here should be:
`
[ivy.array([[1],[3]]),ivy.array([[2],[4]])]
`

File: ivy/container/general.py [hunk 3]
```diff
@@ -364,6 +364,43 @@ def static_unstack(
...
             List of arrays, unpacked along specified dimensions, or containers
             with arrays unpacked at leaves
+
+        Examples
+        --------
+        With one :code:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=ivy.array([[[1, 2], [3, 4]], [[5, 6], [7, 8]]]), 
+                            b=ivy.array([[[9, 10], [11, 12]], [[13, 14], [15, 16]]]))
+        >>> >>> y = ivy.Container.static_unstack(x, axis=0)
```
Comment (JLU-Neal) [inline ivy/container/general.py:374]: redundant ">>>"

File: ivy/container/general.py [hunk 4]
```diff
@@ -364,6 +364,43 @@ def static_unstack(
...
+        >>> print(y)
```
Comment (JLU-Neal) [inline ivy/container/general.py:375]: check the test docstring check, some errors are raised

File: ivy/container/general.py [hunk 5]
```diff
@@ -364,6 +364,43 @@ def static_unstack(
...
+            a: ivy.array([[5, 6],
+                         [7, 8]]),
+             b: ivy.array([[13, 14],
+                          [15, 16]])
+        }]
+
+        >>> x = ivy.Container(a=ivy.array([[[1, 2], [3, 4]], [[5, 6], [7, 8]]]),
+                            b=ivy.array([[[9, 10], [11, 12]], [[13, 14], [15, 16]]]))
+        >>> y = ivy.Container.static_unstack(x, axis=1, keepdims=True)
+        >>> print(y)
```
Comment (JLU-Neal) [inline ivy/container/general.py:391]: make sure the output value is correct

File: ivy/container/general.py [hunk 6]
```diff
@@ -413,6 +450,24 @@ def unstack(
         -------
         ret
             Containers with arrays unpacked at leaves
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([[[1, 2], [3, 4]], [[5, 6], [7, 8]]]),
+                            b=ivy.array([[[9, 10], [11, 12]], [[13, 14], [15, 16]]]))
+        >>> y = ivy.Container.unstack(x, axis=0)
+        >>> print(y)
```
Comment (JLU-Neal) [inline ivy/container/general.py:459]: make sure the output value is correct

File: ivy/functional/ivy/general.py [hunk 7]
```diff
@@ -1377,6 +1379,57 @@ def unstack(
...
+    >>> x = ivy.array([[[1, 2], [3, 4]], [[5, 6], [7, 8]]])
+    >>> y = ivy.unstack(x, axis=0)
+    >>> print(y)
+    [ivy.array([[1, 2],
+                [3, 4]]), ivy.array([[5, 6],
+                [7, 8]])]
+
+    >>> x = ivy.array([[[1, 2], [3, 4]], [[5, 6], [7, 8]]])
+    >>> y = ivy.unstack(x, axis=1, keepdims=True)
+    >>> print(y)
```
Comment (JLU-Neal) [inline ivy/functional/ivy/general.py:1395]: Similarly, please make sure all the output values are correct.

**Review by JLU-Neal — CHANGES_REQUESTED** [review]
The test-docstrings test is not passed, please check the details and make sure all the output values are correct.

**Review by JLU-Neal — APPROVED** [review]
lgtm!

**Pull request merged**